### ISSUE TITLE
Clicking on Collection by `username` redirects to blank page

### ISSUE THREAD

**Issue by PiyushChandra17** [processing/p5.js-web-editor#2611]
### p5.js version

2.9.2

### What is your operating system?

Mac OS

### Web browser and version

Chrome and 117.0.5938.132

### Actual Behavior

Clicking on Collection by `username` redirects to blank page with a horizontal line on top 

https://github.com/processing/p5.js-web-editor/assets/47579287/4bc82c79-a86b-4fae-8c94-f544b0840ca9

### Expected Behavior

It should redirect logged in user to their own created sketches page

### Steps:
1. Open the editor web app
2. Click on hello! username
3. Click on my collections
4. Click on collection name in table
5. Click on collection by `username`
6. Verify it redirects to blank page

**Comment by mhsh312** [comment]
The error is in the following code:

https://github.com/processing/p5.js-web-editor/blob/c44809467112938f411fde442310297d1a944872/client/modules/User/components/Collection.jsx#L299-L303

The `<Link>` here takes the user to `http://<hostname>/<username>/collections/http://<hostname>/<username>/sketches` while it should take them to `http://<hostname>/<username>/sketches`. 

Changing the Link to the following fixes the issue:
`<Link to="/sketches">{owner.username}</Link>`

Before:
![image](https://github.com/processing/p5.js-web-editor/assets/135870090/739e4d8e-6d6f-4af1-bff3-9119510289ed)

After:
![image](https://github.com/processing/p5.js-web-editor/assets/135870090/7a82f303-8648-460e-9a38-ab373a371581)


Please assign this issue to me so I can make a PR.

**Comment by lindapaiste** [comment]
@mhsh312 You are exactly right about the problem and the solution.

@raclim I fixed this months ago in #2253, though it seems I didn't mention it in the PR notes.  Can we get this merged?  It's really frustrating to see people reporting errors that should have been fixed a long time ago.

https://github.com/processing/p5.js-web-editor/blob/1bca271d7333f0f82d4453ada55bb17a55e233d8/client/modules/User/components/CollectionMetadata.jsx#L86-L89
